third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -2725,8 +2725,12 @@ func resourceContainerClusterStateImporter(d *schema.ResourceData, meta interfac
 		project = parts[0]
 		location = parts[1]
 		clusterName = parts[2]
+	case 6:
+		project = parts[1]
+		location = parts[3]
+		clusterName = parts[5]
 	default:
-		return nil, fmt.Errorf("Invalid container cluster specifier. Expecting {location}/{name} or {project}/{location}/{name}")
+		return nil, fmt.Errorf("Invalid container cluster specifier. Expecting {location}/{name}, {project}/{location}/{name} or projects/{project}/locations/{location}/clusters/{name}")
```

Comment:
Can we use parseImportId here now instead of these case statements?

Reply:
Yeah that should be easier. I tended to lean towards the case statements if we weren't setting some of the fields in state (parseImportId automatically sets state)

Setting project, region, etc shouldn't cause any issues though

Reply:
Oh, I realized why I thought this was an issue!

Container cluster allows the user to specify either `location` or `zone`(deprecated) or `region` (deprecated)

We have removed zone and region in 3.0.0 though, so we should be good to go